.mailmap

```diff
@@ -125,6 +126,7 @@ Wojciech Karpiel <w.karpiel@o2.pl>                 Wojciech Karpiel <w.karpiel@a
 Chantal Keller < >                                 ckeller
 Florent Kirchner <fkirchne@gforge>                 fkirchne <fkirchne@85f007b7-540e-0410-9357-904b9bb8a0f7>
 Florent Kirchner <fkirchne@gforge>                 kirchner <kirchner@85f007b7-540e-0410-9357-904b9bb8a0f7>
+Floriant Angeletti < >                             Octachron
```

Comment:
The file is alphabetically ordered by surname, not by first name.

Besides, there is a typo in the first name: Florian Angeletti.

Reply:
typo fixed

Reply:
inserting new values at the end was pretty easy